test/svm/SvmSpoke.HandleReceiveMessage.ts

```diff
@@ -406,4 +409,168 @@ describe("svm_spoke.handle_receive_message", () => {
     routeAccount = await program.account.route.fetch(routePda);
     assert.isFalse(routeAccount.enabled, "Route should be disabled");
   });
+
+  it("Relays root bundle remotely", async () => {
+    // Encode relayRootBundle message.
+    const relayerRefundRoot = crypto.randomBytes(32);
+    const slowRelayRoot = crypto.randomBytes(32);
+    const calldata = ethereumIface.encodeFunctionData("relayRootBundle", [relayerRefundRoot, slowRelayRoot]);
+    const messageBody = Buffer.from(calldata.slice(2), "hex");
+    const message = encodeMessageHeader({
+      version: cctpMessageversion,
+      sourceDomain: remoteDomain.toNumber(),
+      destinationDomain: localDomain,
+      nonce: BigInt(nonce),
+      sender: crossDomainAdmin,
+      recipient: program.programId,
+      destinationCaller,
+      messageBody,
+    });
+
+    // Remaining accounts specific to RelayRootBundle.
+    const rootBundleId = (await program.account.state.fetch(state)).rootBundleId;
+    const rootBundleIdBuffer = Buffer.alloc(4);
+    rootBundleIdBuffer.writeUInt32LE(rootBundleId);
+    const seeds = [Buffer.from("root_bundle"), state.toBuffer(), rootBundleIdBuffer];
+    const [rootBundle] = web3.PublicKey.findProgramAddressSync(seeds, program.programId);
+    // Same 3 remaining accounts passed for HandleReceiveMessage context.
+    const relayRootBundleRemainingAccounts = remainingAccounts.slice(0, 3);
+    // payer in self-invoked SetEnableRoute.
+    relayRootBundleRemainingAccounts.push({
```

Comment:
Does order matter here?

Reply:
Regular accounts have named props, so the client can reorder them as needed, but remaining accounts is plain array, so I believe the ordering here matters.